fhir-persistence-schema/src/main/resources/postgres/delete_resource_parameters.sql

```diff
@@ -40,21 +40,21 @@ BEGIN
 	USING p_logical_resource_id;
 	EXECUTE 'DELETE FROM {{SCHEMA_NAME}}.' || p_resource_type || '_security            WHERE logical_resource_id = $1'
 	USING p_logical_resource_id;
-    EXECUTE 'DELETE FROM {{SCHEMA_NAME}}.' || p_resource_type || '_ref_values WHERE logical_resource_id = $1'
+    EXECUTE 'DELETE FROM {{SCHEMA_NAME}}.' || p_resource_type || '_ref_values          WHERE logical_resource_id = $1'
```

Comment:
still some mixed spaces and tabs in here, though not the end of the world

Reply:
fixed